flake.nix

```
@@ -58,7 +58,8 @@
     };
 
     iohk-nix = {
-      url = "github:input-output-hk/iohk-nix";
+      url = "github:input-output-hk/iohk-nix/angerman/remove-old-envs";
```

This needs to be merged, before we can merge _this_ PR.

```suggestion
      url = "github:input-output-hk/iohk-nix";
```